toolchains/workspace-pnpm/run_in_dir.py

```diff
@@ -31,13 +41,18 @@ def compute_path(arg: str) -> str:
 
     cwd = args.cwd
 
+    dist_path = None
+    if args.package_dir:
```

Comment:
This introduces some overloading here depending on if a `--package-dir` argument is passed in. Do we think this is ok, or should this be a separate toolchain function?